src/layouts/BlogLayout.astro

```diff
@@ -0,0 +1,53 @@
+---
+import '@fontsource/inter';
+import '@fontsource/inter/500.css';
+import '@fontsource/inter/600.css';
+import '@fontsource/inter/700.css';
+import '@fontsource/inter/800.css';
+import '@fontsource/inter/900.css';
+import { config } from '../config';
+import { Navbar } from '../components/Navbar.jsx';
+import { Footer } from '../components/Footer.jsx';
+import { BlogSubscribe } from '../components/BlogSubscribe.jsx';
+
+import '../styles/Theme.css';
+
+const { frontmatter } = Astro.props;
+---
+
+<!doctype html>
```

Comment:
This is a copy of MarkdownLayout.astro.  There might be a good way to share more. 🤷